client/store/index.js

```diff
@@ -77,6 +78,10 @@ const state = getDefaultState();
 
 Vue.use(Vuex);
 
+const vuexLocal = new VuexPersistence({
+  storage: window.localStorage,
```

Comment:
nit: so it still does not store any state on the server?

Reply:
That is correct. Also if a user clears their cache, they will lose their settings. At some point in the future we should persist state to the server but its not a high priority.